### PR TITLE
`Development:` Add bamboo build agent container to atlassian compose file

### PR DESCRIPTION
**Pull request by TheZoker**
#### General

- [ ] I tested **all** changes and their related features with **all** corresponding user types on a test server.
- [ ] This is a small issue that I tested locally and was confirmed by another developer on a test server.
- [x] Language: I followed the [guidelines for inclusive, diversity-sensitive, and appreciative language](https://docs.artemis.ase.in.tum.de/dev/guidelines/language-guidelines/).
- [x] I chose a title conforming to the [naming conventions for pull requests](https://artemis-platform.readthedocs.io/en/latest/dev/guidelines/development-process.html#naming-conventions-for-github-pull-requests).

### Motivation and Context

There is a new bamboo build agent docker image available, that is build from this file:
https://github.com/ls1intum/Artemis-Local-Setup-Docker/blob/main/bamboo-build-agent/Dockerfile

This will replace the custom bamboo build for C and swift in the future. 
The PR adds the needed parts to the `atlassian.yml` docker compose file, so the build agent service is created a long site the bamboo server.

### Description

Adds the custom bamboo build image compose definition to the `atlassian.yml` file.

### Steps for Testing
1. Use the docker compose file to run the atlassian services. 
2. Go to this path, to approve the build agent `/admin/agent/configureAgents!doDefault.action` under `Remote agents` and then `Agent authorisation`
3. Approve the request
4. Wait some minutes until the build agent has finished its setup
5. Look for this line under the build agents executables (it should be automatically be detected):
image
6. Now bamboo can use this agent to run docker tasks (like build, publish images etc.)

#### Code Review
- [ ] Review 1
- [ ] Review 2
#### Manual Tests
- [ ] Test 1
- [ ] Test 2

### Test Coverage

| Class/File | Branch | Line |
|------------|-------:|-----:|

### CODE REVIEW

**Comment by Mtze** [comment]
Please change the merge destination to the deployment develop branch. Other than that - it is looking good to me

**Comment by TheZoker** [comment]
@Mtze Done!

**Review by Mtze — APPROVED** [review]

File: src/main/docker/atlassian.yml [hunk 1]
```diff
@@ -37,6 +37,18 @@ services:
         networks:
             artemis:
                 ipv4_address: 172.20.0.4
+    bamboo_build_agent:
+        container_name: artemis_bamboo_build_agent
+        image: ghcr.io/ls1intum/artemis-bamboo-build-agent:8.1.3
```
Comment (krusche) [inline src/main/docker/atlassian.yml:42]: 8.2.5?

**Review by krusche — CHANGES_REQUESTED** [review]
Should we not use 8.2.5 now?

File: src/main/docker/atlassian.yml (hunk 1 above)
Comment (Mtze) [reply src/main/docker/atlassian.yml:42]: you're right!

Comment (Mtze) [inline src/main/docker/atlassian.yml:42]:
```suggestion
        image: ghcr.io/ls1intum/artemis-bamboo-build-agent:8.2.5
```

**Review by Mtze — CHANGES_REQUESTED** [review]

**Review by dangcosta — APPROVED** [review]
Considering that the change of version requested by Stephan was done, the rest looks good to me

**Pull request closed without merging**